examples/notebooks/creating_a_problem.ipynb

```diff
@@ -0,0 +1,194 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameters = pybop.Parameters(\n",
+    "    pybop.Parameter(\n",
+    "        \"Positive electrode thickness [m]\",\n",
+    "        prior=pybop.Gaussian(7.56e-05, 0.05e-05),\n",
+    "        bounds=[65e-06, 10e-05],\n",
+    "    ),\n",
+    "    pybop.Parameter(\n",
+    "        \"Positive particle radius [m]\",\n",
+    "        prior=pybop.Gaussian(5.22e-06, 0.05e-06),\n",
+    "        bounds=[2e-06, 9e-06],\n",
+    "    ),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The third component is the dataset. This could be an experimental dataset or a simulation. In this example, we just use simulation data by running model.predict() for a 1C discharge, similar to the creating_a_problem example. To pass it to the problem, we need to extract the voltage, time and current data from the simulation results and define it as the dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "t_eval = np.arange(0, 3700, 1)\n",
+    "solution = model.predict([], t_eval)  # No inputs i.e []\n",
+    "dataset = pybop.Dataset(\n",
+    "    {\n",
+    "        \"Time [s]\": np.arange(0, len(solution[\"Voltage [V]\"].data), 1),\n",
+    "        \"Current function [A]\": solution[\"Current [A]\"].data,\n",
+    "        \"Voltage [V]\": solution[\"Voltage [V]\"].data,\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setting up the problem\n",
+    "With all parts setup, we can now make a problem. Therefore, we use pybop.FittingProblem() to define a simple fitting problem and the model, parameters and dataset defined before as inputs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "problem = pybop.FittingProblem(model, parameters, dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Testing the problem\n",
+    "To verify our problem, we solve assign a sum of squared error cost function to it and solve it. \n",
+    "The solution can be compared to the values in the parameter_set defined in the beginning."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(array([7.62324125e-05, 5.20703504e-06]), inf)\n",
+      "7.67500163945363e-05\n",
+      "5.310487559937693e-06\n"
+     ]
+    }
+   ],
+   "source": [
+    "cost = pybop.SumSquaredError(problem)\n",
+    "optim = pybop.Optimisation(cost)\n",
+    "result = optim.run()\n",
+    "print(result)\n",
+    "print(parameter_set[\"Positive electrode thickness [m]\"])\n",
+    "print(parameter_set[\"Positive particle radius [m]\"])"
+   ]
+  }
+ ],
+ "metadata": {
```

Comment:
It would be great to have a few concluding thoughts here. A summary of the notebook and a few points on the FittingProblem itself.